src/Microsoft.Azure.WebJobs.Extensions.Dapr/Services/DaprServiceClient.cs

```diff
@@ -32,6 +34,34 @@ class DaprServiceClient
             this.defaultDaprAddress = GetDefaultDaprAddress(nameResolver);
         }
 
+        private static async Task<HttpResponseMessage> DaprHttpCall(Func<Task<HttpResponseMessage>> httpCall)
+        {
+            try
+            {
+                var response = await httpCall();
+                await ThrowIfDaprFailure(response);
+
+                return response;
+            }
+            catch (HttpRequestException ex) when (ex.InnerException is SocketException socketException)
+            {
+                if (socketException.SocketErrorCode == SocketError.ConnectionRefused)
+                {
+                    throw new DaprSidecarNotPresentException(HttpStatusCode.ServiceUnavailable, "ERR_DAPR_SIDECAR_DOES_NOT_EXIST", "Dapr sidecar is not present. Please follow this link (https://docs.dapr.io/operations/troubleshooting/common_issues/) to debug the issue with dapr.", ex);
```

Comment:
Shall we point them to our docs instead? From a functions PoV we should document when the sidecar might not be present.

Reply:
Also better to use an aka.ms link.

Reply:
Do we have docs available? we can add a TODO for now, if docs isn't ready.

Reply:
You can use this https://aka.ms/azure-functions-dapr-sidecar-missing for now, once docs are ready we can update the link.